docs/source/conf.py

```diff
@@ -289,6 +289,7 @@ def setup(app):
     r"https://github.com/[^/]*$",  # too many github usernames / searches in changelog
     "https://github.com/jupyterhub/jupyterhub/pull/",  # too many PRs in changelog
     "https://github.com/jupyterhub/jupyterhub/compare/",  # too many comparisons in changelog
+    "https://schema.jupyter.org/jupyterhub/.*",  # schemas are not published
```

Comment:
(Off topic: _should_ they be published?)

Reply:
Yes, probably. But the schemas repo isn't quite set up yet

Reply:
added "yet"